Lombiq.Analyzers.PowerShell/Invoke-Analyzer.ps1

```diff
@@ -0,0 +1,86 @@
+param(
```

Comment:
The Windows build on GHA seems to be a lot slower than usual now.

Reply:
What do you mean? Build looks to be about the same as other in other branches:
![image](https://user-images.githubusercontent.com/4007293/178254509-6ac5ac84-603c-4fd6-aef0-c80bf08641fc.png)

Reply:
Yes, but test execution is consistently significantly slower. I have no idea what would cause this, but the pattern is there.

Reply:
Yeah, looks like calling `dotnet build $SolutionFileName$ --no-incremental --nologo --warnaserror --consoleLoggerParameters:NoSummary --verbosity:quiet -p:TreatWarningsAsErrors=true -p:RunAnalyzersDuringBuild=true` from inside the unit test takes an ungodly amount of time on Windows. [The same is true for OSOE-110 as well.](https://ci.lombiq.com/buildConfiguration/OrchardExtensions_OSOCE_Developer_PullAndBuild/1060835?hideProblemsFromDependencies=false&hideTestsFromDependencies=false&expandBuildChangesSection=true&expandBuildProblemsSection=true)
So what should we do? It seems to run fast enough on Linux, maybe skip the test on Windows? `DirectScriptInvocationShouldDisplayWarnings` and the GH action for `Utility-Scripts` would still run on Windows so we wouldn't lose too much coverage.

Reply:
I have decided to scrap this. Instead of unit tests, I have created a reusable workflow in our GHA repo for testing expected analyzer violations.

Reply:
I've just seen your previous message, but sounds good then.